PennMobile/DiningCell2.swift

```diff
@@ -0,0 +1,267 @@
+//
+//  DiningCell.swift
+//  PennMobile
+//
+//  Created by Josh Doman on 3/7/17.
+//  Copyright © 2017 PennLabs. All rights reserved.
+//
+
+//import UIKit
+//
+//class DiningCell2: UITableViewCell {
```

Comment:
Don't leave commented out code, if it's not needed take it out, we can always get it back in version control